sdk/python/pulumi_eks/_inputs.py

```diff
@@ -289,7 +289,7 @@ class ClusterNodeGroupOptionsArgsDict(TypedDict):
 
         Note: Given the inheritance of auto-generated CF tags and `cloudFormationTags`, you should either supply the tag in `autoScalingGroupTags` or `cloudFormationTags`, but not both.
         """
-        bootstrap_extra_args: NotRequired[str]
+        bootstrap_extra_args: NotRequired[pulumi.Input[str]]
```

Comment:
Merging https://github.com/pulumi/pulumi-eks/pull/1414 & https://github.com/pulumi/pulumi-eks/pull/1415 ontop of each other dropped those changes.
This adds is back now, work tree is clean again

Reply:
Are we missing CI check guarantees that the worktree should be clean possibly?

Reply:
We're merging those into a feature branch that's only built when cutting an alpha release or when triggered manually. It's checking it as part of those workflows